zcash_transparent/CHANGELOG.md

```diff
@@ -10,6 +10,32 @@ workspace.
 
 ## [Unreleased]
 
+### Added
+- `zcash_transparent`:
+  - `builder`:
+    - `Coinbase` marker type
+    - `impl Authorization for Coinbase`
+    - `impl MapAuth<Coinbase, Authorized> for Coinbase`
+    - `impl TransparentAuthorizingContext for Coinbase`
+    - `TransparentBuilder::build_coinbase`
+    - `std::error::Error for Error`
+  - `bundle`:
+    - `Outpoint::null`
+    - `TxIn::<builder::Coinbase>::coinbase`
+  - `coinbase`:
+    - `MAX_COINBASE_SCRIPT_LEN`
+    - `MIN_COINBASE_SCRIPT_LEN`
+    - `MAX_COINBASE_HEIGHT_LEN`
+    - `MAX_MINER_DATA_LEN`
+    - `MinerData` struct
+    - `impl Default for MinerData`
+    - `impl TryFrom<&[u8]> for MinerData`
+    - `impl AsRef<[u8]> for MinerData`
+    - `Error` enum
+    - `impl fmt::Display for Error`
+    - `impl std::error::Error for Error`
+  - Optional dependency on `serde` under the implicit `serde` feature.
```

Comment:
We do not use implicit feature flags. Define a `serde` feature flag that enables `dep:serde` and is documented similarly to `transparent-inputs` (so it shows up in rustdoc renders).